docs/run_test/test_table.rst

```diff
@@ -11,12 +11,153 @@ Table of Test Cases
     |:construction:| WIP |:construction:|
 
 .. list-table::
-    :widths: 5 30 5 10 10 10
+    :widths: 5 5 25 5 10 10 10
     :header-rows: 1
 
     * - Index
-      - LISA TestName
+      - Test Suite Name
```

Comment:
Can this file be generated automatically?

Reply:
Yes. This is generated by table_gen.py.